web/src/main/webapp/script.js

```diff
@@ -12,89 +12,81 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+function searchMe() {
+  var input, filter, stockList, stockListItem, item, txtValue;
+  input = document.getElementById("myInput");
+  filter = input.value.toUpperCase();
+  stockList = document.getElementById("stock-list");
+  stockListItem = stockList.getElementsByTagName("li"); 
+   for (var i = 0; i < stockListItem.length; i++) {
+    item = stockListItem[i];
+    txtValue = item.textContent || item.innerText;
```

Comment:
`const textValue`

Reply:
Changing them to `const` breaks it but I am not sure why